templates/jenkins-run.erb

```diff
@@ -55,7 +55,9 @@ JAVA_CMD="$JENKINS_JAVA_CMD $JENKINS_JAVA_OPTIONS -DJENKINS_HOME=$JENKINS_HOME -
 PARAMS=()
 PARAMS+=("--logfile=/var/log/jenkins/jenkins.log")
 PARAMS+=("--webroot=/var/cache/jenkins/war")
+<%- if scope['jenkins::systemd_type'] == 'simple' -%>
```

Comment:
Shouldn't this be `forking`?
```suggestion
<%- if scope['jenkins::systemd_type'] == 'forking' -%>
```

Perhaps you could include a test for it in the `with jenkins version x.y` tests too.